helm/daemon/keygen.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -u
+# Set the prefix to the node name (like "seed" or "block-producer")
+# And set count as the number of keys of this type to generate (e.g. "./keygen.sh seed 6" produces 6 keys, "seed-1" through "seed-6")
+PREFIX=$1
+COUNT=$2
+
+for i in $(seq 1 ${COUNT}); do
+
+        NODE="${PREFIX}-${i}"
+
+        export MINA_PRIVKEY_PASS=$(pwgen --no-vowels --secure --ambiguous 64 1)
+        export MINA_LIBP2P_PASS=$(pwgen --no-vowels --secure --ambiguous 64 1)
+
+        KEY="${NODE}-key"
+        PUB="${NODE}-key.pub"
+        LIBP2P="${NODE}-libp2p"
+        PEERID="${NODE}-libp2p.peerid"
+        YAML="${NODE}.yaml"
+
+        echo "Generating keys for ${NODE}"
+        mina advanced generate-keypair --privkey-path ${KEY} 2> /dev/null
```

Comment:
docker image would not require users to have mina installed on their system. It would require docker though 🌚

Reply:
yeah we have run versions like that before but the startup time in docker makes it slow to generate a large volume of keys. Many better ways to handle + port this going forward, the only thing I think worth keeping is the interface with the chart (one secret per chart deployment, or even one secret to many deployments)